### ISSUE TITLE
pornmedium site request

### ISSUE THREAD

**Issue by Stereophobic** [dobbelina/repository.dobbelina#1238]
### Prerequisites

- [X] I have searched the issue tracker to avoid opening a duplicate issue.

### Description

Would be a great add.

### If it's a site addition request, add url below

https://pornmedium.com/

### URL to DMCA information

https://pornmedium.com/DMCA

**Comment by Gujal00** [comment]
A site with a single page!

**Comment by ErosVece** [comment]
@Gujal00, mate....don't drink too much 🤣 

![image](https://github.com/dobbelina/repository.dobbelina/assets/37535332/84bb544e-e669-425e-a1cd-bac39813d9c4)

**Comment by ErosVece** [comment]
Yes, your request is fine, the site is fine....we only need some time to work on it :)

Blame Stable Diffusion and other AI stuff, it's too much fun to play with it :)
